reporting/reporting/Stored Procedures/DimMonthData.sql

```diff
@@ -0,0 +1,21 @@
+﻿
+
+CREATE PROCEDURE [dbo].[DimMonthData]
+
+AS
+BEGIN
+	
+	DECLARE @StartDate date= '20100101'
+
+	WHILE @StartDate <= '20151231'
+		BEGIN
```

Comment:
@Sreenitha-Sampath Adjust indentation of while block.

Reply:
I have removed the blank lines and adjusted the indentatin